ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/Block.java

```diff
@@ -57,18 +59,26 @@ public void writeTo(final RLPOutput out) {
     out.startList();
 
     header.writeTo(out);
-    body.writeTo(out);
+    out.writeList(body.getTransactions(), Transaction::writeTo);
+    out.writeList(body.getOmmers(), BlockHeader::writeTo);
+    body.getWithdrawals().ifPresent(withdrawals -> out.writeList(withdrawals, Withdrawal::writeTo));
+    body.getDeposits().ifPresent(deposits -> out.writeList(deposits, Deposit::writeTo));
```

Comment:
Why have we duplicated the contents of BlockBody.writeTo() here?

Reply:
This was necessary when SSZ was in the picture, will try to revert.